src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -583,37 +598,44 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = (props) => {
           <>
             <div className="input-container">
               <h4 className="italic" style={{ fontSize: "20px" }}>
-                Number of rows
+                Number of columns
               </h4>
               <input
                 disabled={disable}
                 defaultValue={1}
-                name="rows"
+                name="columns"
                 type="number"
                 ref={register}
-                onChange={(e) => {
-                  setValue("rows", parseInt(e.target.value));
-                }}
                 className="align-left"
-                placeholder={`Number of grid rows`}
+                placeholder={`Number of grid columns`}
               />
               {errors.name ? (
                 <span className="input-error">{errors.name.message}</span>
               ) : null}
             </div>
             <div className="input-container">
               <h4 className="italic" style={{ fontSize: "20px" }}>
-                Number of columns
+                Number of rows
               </h4>
+              <div>
+                Not editable. The number of rows is derived from the number of
+                specified columns and the width:height ratio of the party map,
+                to keep the two aligned.
+              </div>
               <input
-                disabled={disable}
-                defaultValue={1}
-                name="columns"
+                disabled={true}
+                defaultValue={rows ?? 1}
+                value={rows ?? 1}
+                name="rows"
                 type="number"
                 ref={register}
-                className="align-left"
-                placeholder={`Number of grid columns`}
+                onChange={(e) => {
+                  setValue("rows", parseInt(e.target.value));
+                }}
```

Comment:
Since this is seemingly always disabled now, does the onChange do anything anymore? When will it change? (and potentially also, does `input` make sense to use anymore if it will never change?)

Reply:
The whole component is now removed because the calculation is happening in the Map.tsx and it's no longer necessary.